### PR TITLE
把問卷照片改成非必填，問券類型 `type` 改成必填

### PR DESCRIPTION
**Pull request by Neilxx**
solve issue #66

### CODE REVIEW

File: src/lib/imageUpload.ts [hunk 1]
```diff
@@ -35,7 +35,7 @@ export const useImageUpload = () => {
     uploadedImages.value = uploadedImages.value.filter(image => image.src !== src)
   }
 
-  const imageUploadFormValid = computed(() => uploadedImages.value.length > 0 && !imageUploadState.uploading)
```
Comment (tai271828) [inline src/lib/imageUpload.ts:38]: : DD thank you for finding this out!!

File: src/components/ImageUploadForm.vue [hunk 2]
```diff
@@ -27,7 +25,7 @@
         </v-btn>
 
         <span v-if="uploading && !disableProgressiveUpload">上傳中...</span>
-        <span v-if="valid && !uploading && !error && !disableProgressiveUpload">
+        <span v-if="valid && !uploading && !error && !disableProgressiveUpload && previewImages.length">
```
Comment (tai271828) [inline src/components/ImageUploadForm.vue:28]: I don't get this. Would you mind elaborating a bit more for me? : )

**Review by tai271828 — COMMENTED** [review]
Looks good me except one question needs your help!

File: src/components/ImageUploadForm.vue (hunk 2 above)
Comment (Neilxx) [reply src/components/ImageUploadForm.vue:28]: `valid` used to be `uploadedImages.value.length > 0 && !imageUploadState.uploading` as below, and I remove the `uploadedImages.value.length > 0` part, so now `valid` basically equal `!imageUploadState.uploading`.
If this line not modified, text `上傳成功` will always show, no matter image uploaded or not.

**Review by yanghaochang104 — APPROVED** [review]

**Comment by Neilxx** [comment]
@tai271828 need your help to deploy to development site. Thanks 🫶

**Comment by diaoga** [comment]
關於此PR的延伸issue #70

**Pull request merged**